packages/caravan-clients/src/types.ts

```diff
@@ -0,0 +1,45 @@
+export interface UTXO {
+  txid: string;
+  vout: number;
+  value: number;
+  status: {
+    confirmed: boolean;
+    block_time: number;
```

Comment:
This UTXO interface was already existing and I have just moved it to another file. We can include block height if needed but can you please elaborate on where do we use it further in context of client or health package ?? I didn't get this.

Reply:
I'll have to think about it some more, but I would think that some of the health APIs where you're calculating over time or frequency for example, you likely care more about the distribution across block height vs blocktime. But maybe the way this is used it doesn't matter as much. Generally I've just tried to avoid block time in the past as it's inherently inaccurate for most use cases (block timestamps have a buffer of around 2 hours that they're allowed to be off I believe)